controllers/nodefeaturediscovery_controls.go

```diff
@@ -1,11 +1,11 @@
 /*
-Copyright 2020 The Kubernetes Authors.
+Copyright 2021.
```

Comment:
Should we keep the same "author". And more, use the same copyright statement in all of the files?